prac_10/wiki.py

```diff
@@ -0,0 +1,10 @@
+import wikipedia
+
+try:
+    search = input("Search: ")
+    while search != '':
+        web_page = wikipedia.page(search)
+        print("\t-{}-\n{}\nVisit page at {}".format(web_page.title, web_page.summary, web_page.url))
+        search = input("Search: ")
+except wikipedia.exceptions.DisambiguationError as e:
+    print(e.options)
```

Comment:
The only issue I can see is that the try-except statement should be **inside** the while loop, the way it currently is the program will still exit on a DisambiguationError, and won't allow the user to further specify their search.  Apart from that issue everthing looks good.